server/rogue/client.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const EventEmitter = require('events');
+const camelCase = require('camelcase');
```

Comment:
Lodash, which we already include in this library, has a [camelCase](https://lodash.com/docs/4.17.10#camelCase) helper that might work here.